### DISCUSSION TITLE
Inherance in github teams

### DISCUSSION THREAD

**Discussion by faisalm5** [community/community#79110] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm have set org/repo level teams in github and hoped that it would support adding teams within teams with inheritance. However, I've noticed that team inheritance doesn't work, and I have to add same people in multiple teams. Has anyone come across this issue and whats your advise?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @faisalm5 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
